templates/server.conf.j2

```diff
@@ -31,6 +36,9 @@ cipher {{ openvpn_cipher }}
 # most systems, the VPN will not function unless you partially or fully disable
 # the firewall for the TUN/TAP interface.
 dev {{ openvpn_dev }}
```

Comment:
Why do you leave the non-ipv6 options outside of a conditional? Does this work?

Reply:
Yes, this works